runtime/oti/jvmtiInternal.h

```diff
@@ -91,15 +91,15 @@ typedef struct J9JVMTIObjectTag {
 	jlong tag;
 } J9JVMTIObjectTag;
 
-#define J9JVMTI_WATCHED_FIELD_ACCESS_BITS_PER_FIELD ((UDATA)2)
+#define J9JVMTI_WATCHED_FIELD_BITS_PER_FIELD 2
 #define J9JVMTI_WATCHED_FIELDS_PER_UDATA \
-	((sizeof(UDATA) * 8) / J9JVMTI_WATCHED_FIELD_ACCESS_BITS_PER_FIELD)
+	((sizeof(UDATA) * 8) / J9JVMTI_WATCHED_FIELD_BITS_PER_FIELD)
 #define J9JVMTI_CLASS_REQUIRES_ALLOCATED_J9JVMTI_WATCHED_FIELD_ACCESS_BITS(clazz) \
 	((clazz)->romClass->romFieldCount > J9JVMTI_WATCHED_FIELDS_PER_UDATA)
 #define J9JVMTI_WATCHED_FIELD_ARRAY_INDEX(index) \
-	(((UDATA)(index)) / J9JVMTI_WATCHED_FIELDS_PER_UDATA)
+	(((index)) / J9JVMTI_WATCHED_FIELDS_PER_UDATA)
```

Comment:
some extra brackets around (index) here, which could be removed, although they aren't harmful.